app/controllers/projects_controller.rb

```diff
@@ -38,7 +37,9 @@ def create
 
   # PATCH/PUT /projects/1
   def update
-    if @project.update(project_params)
+    authorize @project, :update?
+    @project = update_project.project
```

Comment:
You don't need this line

Reply:
I use it to show errors if project was not updated